ansible/monitoring-ceph.yml

```diff
@@ -29,5 +28,6 @@
     monasca_region_name: RegionOne
     monasca_service_type: monitoring
     monasca_log_level: INFO
+    monasca_agent_system_only: true
```

Comment:
I guess you might be looking at this, but you'll need to set `monasca_checks` on the node you want to run the ceph plugin on. Example: https://gitlab.ecdf.ed.ac.uk/stackhpc/ecdf-kayobe-config/merge_requests/5/diffs

Reply:
Surprisingly - wasn't necessary. Seem to pick up the plugins anyway:
```
Successfully initialized checks: ['load', 'network', 'process', 'ntp', 'ceph', 'memory', 'disk', 'cpu']
```